### DISCUSSION TITLE
Nuget vs pac cli tooling version

### DISCUSSION THREAD

**Discussion by ciaomatteo** [microsoft/powerplatform-build-tools#783] [Q&A] [answered]
Dear pac cli team
What version of tools like solution packager, configuration migration tool and package deployer is pac cli using? Is it the same version as provided by NuGet?
Thanks for your advice.

**Comment by petrochuk** [comment]
if your question about `pac tool` command then it is always latest from Nuget.org

**Reply by ciaomatteo** [reply]
Thanks, that helps. And solution packager provided by Nuget (Microsoft.CrmSdk.CoreAssemblies) is not the same as "pac solution pack"?

**Reply by petrochuk** [reply, marked as answer]
In case of `pac solution pack` there might be slight difference with latest versions but in general we try to update to all latest versions of each dependency. We also mention most version updates in "Misc" section of [Release notes](https://www.nuget.org/packages/Microsoft.PowerApps.CLI#releasenotes-body-tab)
